shared/util/feature-flags.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+type FeatureFlag = {
+  tracker: 'v1' | 'v2'
+}
+let ff: FeatureFlag = {
```

Comment:
also we want v2 to only be for us right, so we need some kind of example of this. either process.env or from local-debug or something